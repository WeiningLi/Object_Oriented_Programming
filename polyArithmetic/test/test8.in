create a 5/45 2 2/3 2 1/7 0
print a 
create b 3/5 10 4/5 2 4/7 0
print b
create c 0/1 0
print c
subtract a b
add a b
add a c
add c a
subtract a c
subtract c a
multiply a b
multiply a a
multiply b b
divide b b
divide a b
divide b a
quit
