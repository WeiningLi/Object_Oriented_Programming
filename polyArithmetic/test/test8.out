(2/3)x^2 + (1/7)
(3/5)x^10 + (4/5)x^2 + (4/7)
0
(-3/5)x^10 + (-2/15)x^2 + (-3/7)
(3/5)x^10 + (22/15)x^2 + (5/7)
(2/3)x^2 + (1/7)
(2/3)x^2 + (1/7)
(2/3)x^2 + (1/7)
(-2/3)x^2 + (-1/7)
(2/5)x^12 + (3/35)x^10 + (8/15)x^4 + (52/105)x^2 + (4/49)
(4/9)x^4 + (4/21)x^2 + (1/49)
(9/25)x^20 + (24/25)x^12 + (24/35)x^10 + (16/25)x^4 + (32/35)x^2 + (16/49)
Quotient: (1)
Remainder: 0
Quotient: 0
Remainder: (2/3)x^2 + (1/7)
Quotient: (9/10)x^8 + (-27/140)x^6 + (81/1960)x^4 + (-243/27440)x^2 + (461721/384160)
Remainder: (1074919/2689120)
